pkg/products/threescale/reconciler.go

```diff
@@ -166,7 +166,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return integreatlyv1alpha1.PhaseFailed, err
 	}
 
-	phase, err = r.ReconcilePullSecret(ctx, r.Config.GetNamespace(), "", installation, serverClient)
+	phase, err = r.ReconcilePullSecret(ctx, r.Config.GetNamespace(), "3scale-pull-secret", installation, serverClient)
```

Comment:
normally string values are defined in constants.

Reply:
Made that change